src/content/blam/tags/readme.md

```diff
@@ -26,7 +26,7 @@ Some tag fields are _references_ to other tags. In tools like [Guerilla][], thes
 When tags are compiled into a map, their references are converted into pre-calculated pointers. An array of tag paths are still retained in the map but is not used by the game.
 
 ## Blocks
-A _tag block_ field, also known as a _reflexive_, is essentially an array header. The field consists of an item count and an internal pointer to an array of structures of an expected type. An example is the [scenario][] tag containing a block of [vehicles][vehicle]. In visual tag editors, blocks appear as a list of elements which are often editable by adding or removing elements.
+A _tag block_ field, also known as a _reflexive_, is essentially a list of smaller data structures within a tag. An example is the [scenario][] tag containing a block of [vehicles][vehicle]. In visual tag editors, blocks appear as a list of elements which are often editable by adding or removing elements. A block field internally consists of an item count and a pointer to an array of structures of the expected type.
```

Comment:
@gbMichelle you mentioned that arrays are kind of a foreign concept for most users, so I de-prioritized/reworded that info.